mlrun/config.py

```diff
@@ -240,6 +240,13 @@
             "channel": "master",
         },
     },
+    "storage": {
+        # What type of auto-mount to use for functions. Can be one of: none, auto, v3io_cred, v3io_fuse, pvc.
+        # Default is auto - which is v3io_cred when running on Iguazio and none otherwise
+        "auto_mount_type": "auto",
+        # Extra parameters to pass to the mount call (will be passed as kwargs)
+        "auto_mount_params": {},
```

Comment:
We never used a "dynamic" config value like this, did you try and successfully configured this by setting env vars on a pod ? (not by setting the value in unit tests)

Reply:
You are correct - this doesn't work. The config only accepts values to params it already knows about, so using this for kwargs won't work. I've modified this to be a string with a known format that will be parsed by the code.